build.gradle.kts

```diff
@@ -20,14 +20,14 @@ dependencies {
   implementation("com.google.code.gson:gson:2.8.8")
   implementation("org.awaitility:awaitility-kotlin:4.1.0")
   implementation("com.amazonaws:amazon-sqs-java-messaging-lib:1.0.8")
-  implementation("com.amazonaws:aws-java-sdk-sns:1.12.62")
```

Comment:
would have been worth bumping kotlin to 1.5.30 and gradle spring boot plugin too